README.md

```diff
@@ -1,24 +1,216 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+ Budget App
 
-* Ruby version
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 budget App ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Budget App <a name="about-project"></a>
 
-* Database creation
 
-* Database initialization
+This project is about building a mobile web application to manage a person budget. He will provide list of transactions associated with a category, so that he can see how much money he spent against each transacction.To keep data private, user go through authentication process.
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+<div align="center">
 
-* Deployment instructions
+  <img src="./app/assets/images/diagram.png" alt="Logo" />
+  <br/>
 
-* ...
+</div>
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server Side</summary>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/en/">Ruby on Rails</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Splash Screen**
+- **Sign up and log in pages**
+- **Home page (categories page)**
+- **Transactions page**
+- **Add a new category page**
+- **Add a new transaction page**
+- **Use devise for user authentication**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- <a href="https://muneeb.onrender.com/">Live Demo</a>
+- [Video Presentation](https://www.loom.com/share/75079963b12f4b6bba4361d8ca2d2de5?sid=28798262-1b2e-4c45-9f1f-9fe95d38df3b)
```

Comment:
- [ ] `Required`
- [ ] Amazing presentation ✔️ 
- [ ] with a maximum length of 5 minutes. The content of the video should include:✔️ 
- [ ] a description of the project.✔️ 
- [ ] a demo of the project features.✔️ 
- [ ] you should also highlight some interesting piece of code or something you built that you are very proud of. ✖️ 
- [ ] Please highlight some interesting piece of code or something you built that you are very proud of.